libraries/extensions/dora-record/src/main.rs

```diff
@@ -83,11 +89,12 @@ fn main() -> eyre::Result<()> {
     Ok(())
 }
 
-/// Write a row of data into the writer
+/// Write a row of data into the writerr
 fn write_event(
-    writer: &mut FileWriter<File>,
+    writer: &mut StreamWriter<File>,
```

Comment:
Just out of interest: What's the reason for using StreamWriter instead of FileWriter?

Reply:
So the `FileWriter` needs to finish to be readable where as a `StreamWriter` does not, but is less optimized. At first I wanted to make the `StreamWriter` keeps on writing on the same file, but it seems that it is not possible for now.

I think that still `StreamWriter` should be more resilient so I left it.

Reply:
Thanks for clarifying!